packages/platform-browser/src/hydration.ts

```diff
@@ -127,6 +127,17 @@ function provideZoneJsCompatibilityDetector(): Provider[] {
  * passing special functions (from the `HydrationFeatures` set) as arguments to the
  * `provideClientHydration` function.
  *
+ * By default, the [`provideClientHydration()`](api/platform-browser/provideClientHydration) function enables the recommended set of features for the optimal performance for most of the applications. It includes the following features:
+ * * Reconciling DOM hydration (learn more about it [here](guide/hydration)).
+ * * [`HttpClient`](api/common/http/HttpClient) response caching while running on the server and transferring this cache to the client to avoid extra HTTP requests.
```

Comment:
I've added `...` as a link, not sure what the correct deep link would look like (should we add an `<a>` tag there with a specific name?).
```suggestion
 * * [`HttpClient`](api/common/http/HttpClient) response caching while running on the server and transferring this cache to the client to avoid extra HTTP requests. Learn more about data caching [here](...).
```